packages/backend/src/build-disk-image.ts

```diff
@@ -130,6 +130,11 @@ export async function buildDiskImage(build: BootcBuildInfo, history: History): P
       );
       logData += JSON.stringify(buildImageContainer, undefined, 2);
       logData += '\n----------\n';
+      try {
+        fs.writeFileSync(logPath, logData);
```

Comment:
maybe use the promisify version from 'node:fs/promises'. fs.writeFile(

Reply:
Switched to fs.promises.writeFile(), since we're using fs.promises already (could have both imports but can't remove regular import due to use of existsSync()).